src/pinecone_upsert.py

```diff
@@ -1,7 +1,13 @@
 from pinecone import Pinecone
 import json
+from dotenv import load_dotenv, find_dotenv
+from src.pathmaker import env_path
 
-pc = Pinecone(api_key="3b40a01e-51af-4ebc-bf71-fe1e7104fa08")
+env_path = find_dotenv("config/.env")
+load_dotenv(env_path)
+
+pinecone_api_key = os.getenv("PINECONE_API_KEY")
+pc = Pinecone(api_key=pinecone_api_key )
 index = pc.Index("web-scrap-index")
 
 with open('vector1_data_for_pinecone.json', 'r') as f:
```

Comment:
@nivesh1000 make sure that chunks are getting uploaded individually and getting vetorized